packages/grafana-data/src/field/displayProcessor.test.ts

```diff
@@ -5,6 +5,7 @@ import { Field, FieldConfig, FieldType, GrafanaTheme, Threshold, ThresholdsMode
 import { getScaleCalculator, sortThresholds } from './scale';
 import { ArrayVector } from '../vector';
 import { validateFieldConfig } from './fieldOverrides';
+import { formattedValueToString } from '../valueFormats/valueFormats';
```

Comment:
Maybe revert this or add back test?
```suggestion
```

Reply:
yea, I tried to fix another related bug but did not manage. will remove it